atd-vze/src/views/Crashes/Notes.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import { useQuery } from "@apollo/react-hooks";
+import { Card, CardHeader, CardBody, CardFooter, Table} from "reactstrap";
+import moment from "moment";
+import { notesDataMap } from "./notesDataMap";
+import { GET_NOTES } from "../../queries/notes";
+
+// declare a notes component
+const Notes = ({...props }) => {
+  // pull crashid for page
+  const crashId = props.match.params.id;
+
+  // fetch data from database using graphQL query
+  const { loading, error, data, refetch } = useQuery(GET_NOTES, {
+    variables: { crashId },
+  });
+
+  if (loading) return "Loading...";
+  if (error) return `Error! ${error.message}`;
+
+  const tableName = "notes";
+  const keyField = "id";
+  const fieldConfig = notesDataMap[0];
+
+  // function to format date field values
+  function formatDate(field, row) {
+    if (field == "date") {
```

Comment:
double equals can be tricky 👻 
https://www.freecodecamp.org/news/javascript-triple-equals-sign-vs-double-equals-sign-comparison-operators-explained-with-examples/